src/CMakeLists.txt

```diff
@@ -1,10 +1,21 @@
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(EventdEvent REQUIRED libeventd-event>=0.3)
+pkg_check_modules(Eventd REQUIRED libeventd>=0.4)
 pkg_check_modules(EventdPlugin REQUIRED libeventd-plugin>=0.3)
 pkg_check_modules(SystemdJournal REQUIRED libsystemd-journal)
 
+macro(pkg_config_variable PKG VAR OUT)
```

Comment:
Style: The variables should be "out pkg var". My style is to use lowercase for internal variables and uppercase for user-modifiable variables.